### PR TITLE
Add template version check

### PR DESCRIPTION
**Pull request by picklelo**
* Add `pcversion.txt` file in the template - we should update this whenever we change the template.
* If the app was initialized with an old version, ask the user to run `pc init` again.
* Update the poetry project to v0.1.11

### CODE REVIEW

File: pynecone/.templates/web/pcversion.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+0.1.11
```
Comment (Alek99) [inline pynecone/.templates/web/pcversion.txt:1]: What the reason for using a txt file vs having a variable in the constants.py

**Review by Alek99 — CHANGES_REQUESTED** [review]

File: pynecone/.templates/web/pcversion.txt (hunk 1 above)
Comment (Alek99) [reply pynecone/.templates/web/pcversion.txt:1]: Oh wait nvm I get it because if they update pc the web folder could be made from the older one

**Review by Alek99 — APPROVED** [review]
This makes sense to me nicely done clever with the txt file

File: pynecone/.templates/web/pcversion.txt (hunk 1 above)
Comment (picklelo) [reply pynecone/.templates/web/pcversion.txt:1]: Yeah I tried the config way but this seemed easier

**Pull request merged**